bam_masterdata/datamodel/object_types.py

```diff
@@ -8722,6 +8722,46 @@ class Weldment(ExperimentalStep):
         section="Identifiers",
     )
 
+    weld_joint_number = PropertyTypeAssignment(
+        code="EXPERIMENTAL_STEP.WELD_JOINT_NUMBER",
+        data_type="INTEGER",
+        property_label="Joint Number",
+        description="""Consecutive numbering of weld joints of a workpiece or component//Fortlaufende Numerierung von Schweißnähten an Werkstücken und Bauteilen""",
+        mandatory=False,
+        show_in_edit_views=False,
```

Comment:
I think this attr was not properly exported by pybis; this is related with the editability of the property in the ELN, so I am guesisng all of yours are always True, right?

Reply:
I will set a default value to True, so my recommendation is simply to delete this.